### ISSUE TITLE
Focus outline should be visible on links, buttons, forms, inputs

### ISSUE THREAD

**Issue by chrisauclair** [cs50/video50#84]
To make the player keyboard accessible, focus outlines should be visible on all button, link, form, and and input elements. Either reenable the user browser default styles, or create a unified focus outline style. 

It's common to see these outlines disabled by reset CSS, but they're vitally important to keyboard users. If you want to hide them from mouse users, you could follow YouTube's lead and only display them once the user hits `tab`.

**Comment by lukejacksonn** [comment]
This has been implemented in https://github.com/cs50/video50/pull/123. I went ahead and took heed from YouTube as you suggested and have the class `is-tabbing` applied to the body when a keyboard user starts tabbing, making the outlines visible.

**Comment by chrisauclair** [comment]
Looks great!
